static/js/containers/DashboardPage.js

```diff
@@ -430,6 +432,13 @@ class DashboardPage extends React.Component {
         />;
       }
 
+      let couponCard;
+      // $FlowFixMe: thinks program is possibly undefined even though it's used above
```

Comment:
I think I figured this out. It thinks that `program` could change to a null or undefined value by the time the callback is executed. If I define `programId` outside the callback the problem goes away